<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVGClass</class>
 <widget class="QWidget" name="SVGClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVG</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushDebTracer">
     <property name="text">
      <string>Deb Tracer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushDebGeneration">
     <property name="text">
      <string>Deb Generation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushContinuerTracer">
     <property name="text">
      <string>Continu Tracer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushFinGeneration">
     <property name="text">
      <string>Fin Generation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="svg.qrc"/>
 </resources>
 <connections/>
</ui>
